tests/test_process.py

```diff
@@ -1,7 +1,32 @@
-from aiida import orm
+import pytest
+from aiida import engine, orm
 
 
-def test_process_inputs(generate_calc_job_node):
+@pytest.mark.usefixtures("aiida_profile_clean")
+def test_submit_button_widget(multiply_add_process_builder_ready):
+    """Test SubmitButtonWidget with a simple `WorkChainNode`"""
+    from aiida.workflows.arithmetic.multiply_add import MultiplyAddWorkChain
+
+    import aiidalab_widgets_base as awb
+
+    def hook(_=None):
+        pass
+
+    def return_inputs():
+        return multiply_add_process_builder_ready
+
+    widget = awb.SubmitButtonWidget(
+        process_class=MultiplyAddWorkChain, inputs_generator=return_inputs
+    )
+
+    widget.on_submitted(hook)
+
+    # Simulate the click on the button.
+    widget.on_btn_submit_press()
```

Comment:
I think this will trigger the submit and there will be node in the DB.

Reply:
That is correct. Is it a problem that should be fixed?

Reply:
I don't think there is any problem here, just think we can add an assert to test the button is clicked and the process is triggered.

Reply:
done.